drivers/SmartThings/matter-thermostat/src/test/test_matter_thermo_setpoint_limits.lua

```diff
@@ -128,6 +113,10 @@ end
 test.register_coroutine_test(
   "Heat setpoint lower than min",
   function()
+    test.socket.matter:__queue_receive({
```

Comment:
I don't think the `queue_receive` calls you have added here and in the following 5 tests are doing anything at the moment. Since you are only receiving either the min or the max in each case, the setpoint limits are not actually set because we only emit the setpoint limit capability event when both the min and max have been received. So, you can remove these from the following test cases.

Reply:
These test cases need the heating and cooling setpoint limits to be defined in order to verify the functionality of `setHeatingSetpoint` and `setCoolingSetpoint`. Since we removed the `queue_receive` calls from `test_init`, the setpoint limits are not being set for the mock device and so the test cases fail. We could move the `queue_receive` calls back to `test_init`, but then we would also need to check that the capabilities for the heating and cooling setpoint ranges are being sent. If we go that route, then would it make sense to remove the three new test cases added by this PR?

Reply:
Thank you for clarifying, I understand why these are here now! I think what you've done here is good, so you can keep the test cases as they are, including the 3 test cases you've added.